src/Dommel/DommelMapper.cs

```diff
@@ -1486,7 +1486,7 @@ private sealed class PostgresSqlBuilder : ISqlBuilder
         {
             public string BuildInsert(string tableName, string[] columnNames, string[] paramNames, PropertyInfo keyProperty)
             {
-                var sql = string.Format("insert into {0} ({1}) values ({2}) select last_insert_rowid() id",
+                var sql = string.Format("insert into {0} ({1}) values ({2})",

```

Comment:
You just completely removed the select statement for the ID. Didn't you mean to add a `select LASTVAL()` or use a `RETURNING` statement?